tooling/rib-intellij-plugin/build.gradle

```diff
@@ -197,19 +197,22 @@ afterEvaluate {
     }
 }
 
-uploadArchives {
+publishing {
     repositories {
-        mavenDeployer {
-            repository url: "file://" + new File(System.getProperty("user.home"), ".m2/repository").absolutePath
+        maven {
             if (isReleaseBuild() && hasMavenCredentials()) {
-                repository(url: "${config.build.artifactoryUrl}/${pluginsMavenRepositoryBucket}") {
-                    authentication(userName: mavenUser, password: mavenPassword)
+                url = "${config.build.artifactoryUrl}/${pluginsMavenRepositoryBucket}"
+                credentials {
+                    username = mavenUser
+                    password = mavenPassword
                 }
+            } else {
+                url = "file://" + new File(System.getProperty("user.home"), ".m2/repository").absolutePath
             }
         }
     }
 }
-build.dependsOn uploadArchives
+build.dependsOn publishing
```

Comment:
A task called `publishing` doesn't exist. Tbh it's strange that `build` publishes something. Usually `build` assembles all artifacts and runs all checks. Publishing is done separately in most open source libraries that I came across.